docs/users.md

```diff
@@ -28,7 +28,7 @@ As an example, using our `cluster1` PostgreSQL cluster, we would see the followi
 Users and databases can be customized in `spec.users` section in the Custom Resource. Section can be changed at the cluster creation time and adjusted over time. Note the following:
 
 - If `spec.users` is set during the cluster creation, the Operator will not create any default users or databases except for PostgreSQL. If you want additional databases, you will need to specify them.
-- For any users added in `spec.users`, the Operator will create a Secret of the `<clusterName>-pguser-<userName>` format. This Secret will contain the user credentials.
+- For any users added in `spec.users`, the Operator will create a Secret of the `<clusterName>-pguser-<userName>` format (such default naming can be altered with the `spec.users.secretName` option). This Secret will contain the user credentials.
```

Comment:
Do I understand correctly that if you have multiple users specified in `spec.users` and you specify `users.secretName` you will not get secret per user, but instead one secret which holds all passwords?

Reply:
Got it. We have no way to show that some value is the part of an array item in our section-point-option notation, so I changed wording to indicate this is done on per-user basis